src/services/ApiService.js

```diff
@@ -12,12 +12,38 @@ const GetStatus = () =>
     .get(`${getValue("apiRoot")}/status`)
     .then(({ status }) => status === 200);
 
+/**
+ * Builds the url dependant on parameters sent.
+ */
+const BuildEndPoint = props => {
+  const { endPoint, filters } = props;
+  var prevType;
+
+  const url = `https://structuredcontentdev.bcg.ad.bcgov.us${endPoint}${
+    filters.length > 0 ? "?" : ""
+  }${filters
+    .map(item => {
+      const { type, value } = item;
```

Comment:
this function should be re-addressed as part of the larger refactor. I do have some tests that help support this when we are ready.